packages/edit-post/src/components/header/header-toolbar/index.native.js

```diff
@@ -39,25 +39,35 @@ import {
 import styles from './style.scss';
 import { store as editPostStore } from '../../../store';
 
+const shadowStyle = {
+	shadowOffset: { width: 2, height: 2 },
+	shadowOpacity: 1,
+	shadowRadius: 6,
+	elevation: 18,
+};
+
 function HeaderToolbar( {
 	hasRedo,
 	hasUndo,
 	redo,
 	undo,
 	showInserter,
 	showKeyboardHideButton,
-	getStylesFromColorScheme,
 	insertBlock,
 	onHideKeyboard,
-	onOpenBlockSettings,
 	isRTL,
 	noContentSelected,
 } ) {
 	const anchorNodeRef = useRef();
-	const containerStyle = getStylesFromColorScheme(
-		styles[ 'header-toolbar__container' ],
-		styles[ 'header-toolbar__container--dark' ]
-	);
+	const isAndroid = Platform.OS === 'android';
```

Comment:
Nitpick: If we import `Platform` from `@wordpress/element` instead of RN, we could do the following:

```suggestion
	const isAndroid = Platform.isAndroid;
```

Reply:
Nice suggestion! Updated in https://github.com/WordPress/gutenberg/pull/52336/commits/fec5b9567daa99fa327f293e1153bfe545a7fc0c